### ISSUE TITLE
Add the edit/remove button to Project Tasks subpanel

### ISSUE THREAD

**Issue by Mausino** [salesagility/SuiteCRM#532]
Hi,

All subpanels have the edit/remove buttons, but only Project Tasks don't use this extension.

Will it possible add this to Project Tasks subpanel ?

Miky

**Comment by mayerelyashiv** [comment]
Hi Mausino, 
i found how to add edit/remove button to Project Tasks subpanel in Project module:
you have to add this:

```
    'edit_button' => 
  array (
    'vname' => 'LBL_EDIT_BUTTON',
    'widget_class' => 'SubPanelEditButton',
    'module' => 'Project',
    'width' => '3%',
    'default' => true,
  ),
  'remove_button' => 
  array (
    'vname' => 'LBL_REMOVE',
    'widget_class' => 'SubPanelRemoveButton',
    'module' => 'Project',
    'width' => '3%',
    'default' => true,
```

in 
/custom/modules/ProjectTask/metadata/subpanels/Project_subpanel_projecttask.php

code should become:

`
<?php
// created: 2016-04-04 17:11:49
$subpanel_layout['list_fields'] = array (
  'name' => 
  array (
    'vname' => 'LBL_LIST_NAME',
    'widget_class' => 'SubPanelDetailViewLink',
    'width' => '70%',
    'default' => true,
  ),
  'assigned_user_name' => 
  array (
    'type' => 'relate',
    'link' => true,
    'vname' => 'LBL_ASSIGNED_USER_NAME',
    'id' => 'ASSIGNED_USER_ID',
    'width' => '10%',
    'default' => true,
    'widget_class' => 'SubPanelDetailViewLink',
    'target_module' => 'Users',
    'target_record_key' => 'assigned_user_id',
  ),
  'date_start' => 
  array (
    'vname' => 'LBL_DATE_START',
    'width' => '15%',
    'default' => true,
  ),
  'date_finish' => 
  array (
    'vname' => 'LBL_DATE_FINISH',
    'width' => '15%',
    'default' => true,
  ),
    'edit_button' => 
  array (
    'vname' => 'LBL_EDIT_BUTTON',
    'widget_class' => 'SubPanelEditButton',
    'module' => 'Project',
    'width' => '3%',
    'default' => true,
  ),
  'remove_button' => 
  array (
    'vname' => 'LBL_REMOVE',
    'widget_class' => 'SubPanelRemoveButton',
    'module' => 'Project',
    'width' => '3%',
    'default' => true,
  ),
);
`

**Comment by samus-aran** [comment]
Hi guys, would be great to see this fix applied to the core of the module instead of custom.  Feel free to provide a PR for this @mayerelyashiv if possible.

**Comment by Mausino** [comment]
@samus-aran will great if will this patch for this ticket applied after 3 years :) Should it be in milestone 7.10.12  
We have this applied on our instance and it's working fine over many years :)

**Comment by samus-aran** [comment]
Ahhh is the fix the one @Dillon-Brown provided? If so then we can update that branch and merge it in if you have provided confirmation it has worked for year :wink:

**Comment by pribeiro42** [comment]
Assessed and working.

**Comment by Mausino** [comment]
@samus-aran should i please for 4th time that will not forget on this ticket :)

**Comment by Dillon-Brown** [comment]
@Mausino I'll need to take a proper look at #4366 and resolve the conflicts. For this issue I can just create a separate PR so it can be easily merged.

**Comment by Mausino** [comment]
@Dillon-Brown thank you for your pull request.. it's working great 👍 should be this ticket marked with milestone label or next relase label also ?

**Comment by Mausino** [comment]
@Dillon-Brown @samus-aran 

i see that https://github.com/salesagility/SuiteCRM/pull/6690 was merged for 7.8 LTS version.. should be it merged also to 7.11.1 and 7.10+ versions ?

**Comment by Mausino** [comment]
@Dillon-Brown @samus-aran i see that this was merged and should it conntain version 7.11 and 7.10+

Should i close this issue ??? or will merge to LTS

https://suitecrm.com/lts/ by this it looks that  7.8 version will soon death and will great if will corrected the graph also for 7.11 version (because was only relased now)

**Comment by Dillon-Brown** [comment]
@Mausino Yes, i'll create some duplicate PR's for the other branches. Thanks!
